controller/RestTestCenterUsers.php

```diff
@@ -0,0 +1,251 @@
+     *     @OA\Parameter(
+     *       name="testCenter",
+     *       in="path",
+     *       description="The test center id",
+     *       required=true,
+     *       @OA\Schema(
+     *           type="string"
+     *       )
+     *     ),
+     *     @OA\Parameter(
+     *       name="user",
+     *       in="path",
+     *       description="User id",
+     *       required=true,
+     *       @OA\Schema(
+     *           type="string"
+     *       )
+     *     ),
+     *     @OA\Parameter(
+     *       name="role",
+     *       in="path",
+     *       description="Role id",
+     *       required=true,
+     *       @OA\Schema(
+     *           enum={"http://www.tao.lu/Ontologies/TAOProctor.rdf#TestCenterAdministratorRole", "http://www.tao.lu/Ontologies/TAOProctor.rdf#ProctorRole"},
+     *           type="string"
+     *       )
+     *     ),
+     *     @OA\Response(
+     *         response="200",
+     *         description="Remove user from the test center",
+     *         @OA\MediaType(
+     *             mediaType="application/json",
+     *             @OA\Schema(
+     *                 type="object",
+     *                 @OA\Property(
+     *                     property="success",
+     *                     type="boolean",
+     *                     description="`false` on failure, `true` on success",
+     *                 ),
+     *                 example={
+     *                     "success": true
+     *                 }
+     *             ),
+     *         ),
+     *     ),
+     * )
+     */
+    public function delete()
+    {
+        try {
+            $testCenter = $this->getTCFromRequest();
+            $user = $this->getUserFromRequest();
+            $role = $this->getRoleFromRequest();
+
+            $this->returnJson([
+                'success' => $this->getService()->unassignUser($testCenter, $user, $role)
+            ]);
+        } catch (\common_Exception $e) {
+            return $this->returnFailure($e);
+        }
+    }
+
+    /**
+     * TODO:
+     */
+    public function get()
+    {
+        //todo: implement retrieving test center users
+    }
+
+    /**
+     * @return \oat\oatbox\user\User
+     * @throws \common_exception_BadRequest
+     * @throws \common_exception_Error
+     * @throws \common_exception_MissingParameter
+     */
+    private function getUserFromRequest()
+    {
+        $userUri = $this->getParameterFromRequest(self::PARAMETER_USER_URI);
+        return $this->getUserService()->getUserById($userUri);
+    }
+
+    /**
+     * @return \core_kernel_classes_Resource
+     * @throws \common_exception_MissingParameter
+     * @throws \common_exception_NotFound
+     */
+    private function getRoleFromRequest()
+    {
+        $roleUri = $this->getParameterFromRequest(self::PARAMETER_USER_ROLE);
+        return $this->getAndCheckResource($roleUri, 'http://www.tao.lu/Ontologies/generis.rdf#UserRole');
```

Comment:
Are you sure it's not `UserRoles` instead of `UserRole`? I didn't find any constant about that, only this one: `oat\generis\model\user\UserRdf::PROPERTY_ROLES`